app/views/layouts/application.html.erb

```diff
@@ -8,11 +8,9 @@
   </head>
   <body>
     <div class="container">
-      <div>
-        <%= link_to "Add Quote", new_quote_path %>
-        <%= link_to "All Quotes", quotes_path %>
+      <div class="navbar-container">
         <% if current_user %>
-          <%= link_to "Sign Out", destroy_user_session_path, method: :delete %>
+          <%= render "layouts/navbar" %>
```

Comment:
I think that if we're going to be quoting people without their permission, then required login is probably a good idea.

I'm fine with my dumbass statements on the googles, but let's not assume everyone is.